pkg/instance/instance.go

```diff
@@ -29,7 +29,7 @@ import (
 
 type Env interface {
 	BuildSyzkaller(string, string) (string, error)
-	BuildKernel(string, string, string, string, string, []byte) (string, build.ImageDetails, error)
+	BuildKernel(string, string, string, string, string, string, []byte) (string, build.ImageDetails, error)
```

Comment:
It feels like it's already the time to somehow group those `BuildKernel` parameters..